grouper/fe/settings.py

```diff
@@ -25,6 +25,7 @@ def override_timezone(self, value):
     "permission_request_text_help": None,
     "port": 8989,
     "refresh_interval": 60,
+    "shell": [["/bin/bash", "bash"], ["/bin/zsh", "zsh"]],
```

Comment:
interesting. i want to say i've had `patch` work with these requests but haven't confirmed recently. can you recall how you specifically tried to patch settings?